src/typeDefs/mutations.ts

```diff
@@ -157,8 +157,11 @@ export const mutations = gql`
 
     updateLanguage(languageCode: String!): User! @auth
 
-    updateOrganization(id: ID!, data: UpdateOrganizationInput): Organization!
-      @auth
+    updateOrganization(
+      id: ID!
+      data: UpdateOrganizationInput
+      file: String
```

Comment:
Instead of file, naming it image, avatar etc., would be better. For now its type is `String`, but later on it will be changed to `File` I suppose. 

Also, include this one inside the input field too, don't need a seperate argument for it.

Reply:
@xoldyckk I thought the file type is there to denote both image and video files that is. 

But changing its name OR keeping it inside the input type would need changing in a lot of places within multiple resolvers where if(args.file ) check is there and changing the queries and mutations in the admin-portal too in the client, it will require a lot of work. And yk the exam thingy and all.

Reply:
But you are right though the file input could be handle within the `data:` field itself so in the after 18th PR I will do that too.

Reply:
Hmm, yes I guess it can be a lot of work if you'll also need to change the tests, and also make the changes to clients and thier tests as well.